VimeoNetworking/Sources/AuthenticationController.swift

```diff
@@ -564,9 +571,7 @@ final public class AuthenticationController
         {
             try self.setClientAccount(with: account, shouldClearCache: true)
             
-            let accountType: AccountStore.AccountType = (account.user != nil) ? .User : .ClientCredentials
-            
-            try self.accountStore.saveAccount(account, type: accountType)
+            try self.saveAccount(account)
```

Comment:
@ghking https://vimean.atlassian.net/browse/VIM-4978 Here's the JIRA for the refactor.